src/ConnectionFactory/PredisConnectionFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace RedisProxy\ConnectionFactory;
+
+use Predis\Client;
+
+class PredisConnectionFactory implements ConnectionFactory
+{
+    /**
+     * @return Client
+     */
+    public function create(string $host, int $port, float $timeout = 0.0)
```

Comment:
timeout ignorujes?

Reply:
Toto je 1:1 s predchadzajucou verziou. Tam sa predisu tiez nenastavoval timeout, v readme som ani nenasiel ako sa mu nastavuje. Musim pozriet do kodu jedine.

Reply:
skus pls, asi som to zabudol kedze pouzivame len Redis

Reply:
Pridané.